specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/applicationGateway.json

```diff
@@ -1681,6 +1685,10 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "Redirect configuration resource of the application gateway."
         },
+        "firewallPolicy": {
```

Comment:
For routing rules of 'Basic' type we should add a map of new objects.
That new object should be similar to ApplicationGatewayPathRulePropertiesFormat but only have 
`"paths"` and `"firewallPolicy"`. We can have multiple path prefixes with different WAF policies per a single Basic rule.

Reply:
Addressed!